scripts/setup-ubuntu.sh

```diff
@@ -14,17 +14,14 @@
 # limitations under the License.
 
 # Minimal setup for Ubuntu 20.04.
+set -eufx -o pipefail
 
 # Folly must be built with the same compiler flags so that some low level types
 # are the same size.
 export COMPILER_FLAGS="-mavx2 -mfma -mavx -mf16c -masm=intel -mlzcnt"
-BUILD_DIR=_build
-
-set -eufx -o pipefail
-
-# Enter build directory.
-mkdir -p "${BUILD_DIR}"
-cd "${BUILD_DIR}"
+FB_OS_VERSION=v2021.05.10.00
+NPROC=$(getconf _NPROCESSORS_ONLN)
```

Comment:
where are you planning to use this variable?

Reply:
Will be used in presto_cpp project